src/components/ProjectDataTable/index.tsx

```diff
@@ -55,7 +55,7 @@ const columns: ColumnDef<any>[] = [
       const uid = row.original[5].value.value[0];
       return (
         <a
-          href={`https://base-sepolia.easscan.org/attestation/view/${uid}`}
+          href={`https://base.easscan.org/attestation/view/${uid}`}
```

Comment:
link should be `ENV` specific
- when NEXT_PUBLIC_TESTNET === `true` -> base-sepolia
- if non-existent -> `base`